src/header.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/sh
+
+set -eu
+TEMPDIR=$(mktemp -d)
+PREFIX=""
+FORCE=0
+VERBOSE=0
+QUIET=0
+UNPACK_SHELL=""
+
+USAGE="
+usage: $0 [options]
+
+Unpacks an environment packed using pixi-pack
+
+-f, --force                 No error if environment already exists
+-h, --help                  Print this help message and exit
+-o, --output-directory <DIR> Where to unpack the environment
+-s, --shell <SHELL>         Sets the shell [options: bash, zsh, xonsh, cmd, powershell, fish, nushell]
+-v, --verbose               Increase logging verbosity
+-q, --quiet                 Decrease logging verbosity
+"
+# Parse command-line options
+while getopts ":hfvo:s:q" opt; do
```

Comment:
What I have implemented right now only supports separate args passed onto the cli but I believe since both of the `-o` and `-s` take an argument afterwards it should be fine.